### ISSUE TITLE
Upgrading to beta.55 prerender fails

### ISSUE THREAD

**Issue by cmcnicholas** [nuxt-modules/og-image#131]
### Describe the bug

After updating to beta.55 I get the following as nuxt tries to prerender the og-image routes, this wasn't happening before and couldn't see anything in the documentation. Are we expected to specify in nuxt config to opt og-image route out of pre-rendering?

```
✔ Server built in 12685ms
[nitro] ℹ Initializing prerenderer
[nitro] ℹ Prerendering 2 routes
[nitro]   ├─ /__og-image__/font/Inter/700.ttf (11ms)
  │ └── Error: [401] 
[nitro]   ├─ /__og-image__/font/Inter/400.ttf (5ms)
  │ └── Error: [401] 
[nitro]
Errors prerendering:
[nitro]   ├─ /__og-image__/font/Inter/700.ttf (11ms)
  │ └── Error: [401] 
[nitro]   ├─ /__og-image__/font/Inter/400.ttf (5ms)
  │ └── Error: [401] 
[nitro]
```

**Comment by cmcnicholas** [comment]
Tested by optiming the `/__og-image__/` path out of prerendering in nuxt config and it's working

```js
  routeRules: {
    '/__og-image__/**': {
      prerender: false,
    },
  },
```

**Comment by harlan-zw** [comment]
Hi, looks like it's failing when trying to request the Inter font from the Google API. Could you share some details about what environment you're building with?

**Comment by cmcnicholas** [comment]
> Hi, looks like it's failing with an unauthorized HTTP status when trying to request the Inter font from the Google API. Could you share some details about what environment you're building with?

```
------------------------------
- Operating System: Windows_NT
- Node Version:     v20.10.0
- Nuxt Version:     3.9.0
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@9.1.2
- Builder:          -
- User Config:      alias, devtools, nitro, modules, vite, site, robots, linkChecker, basicAuth, runtimeConfig, imports, routeRules, typescript
- Runtime Modules:  @vueuse/nuxt@10.7.1, @pinia/nuxt, @nuxtseo/module, @kgierke/nuxt-basic-auth
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Hi, this should be fixed in nuxt-og-image@3.0.0-rc.36, let me know how you go :)
